lonboard-viz.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>What's this variable for? the S3 path? that makes sense, then convert the next cell to an f string inserting the S3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>